getting-started-with-api/java-sdk/src/main/java/DownloadPseudo.java

```diff
@@ -0,0 +1,71 @@
+import com.smartling.api.files.v2.FilesApi;
```

Comment:
It's a common practice to put classes in packages, e.g. `com.smartling.samples`

Reply:
Will add to packages. I assume this package makes sense: com.smartling.samples.gettingstarted